doc/03-Configuration.md

```diff
@@ -0,0 +1,115 @@
+# Configuration
+
+The configuration is stored in `/etc/icinga-notifications/config.yml`.
+See [config.example.yml](../config.example.yml) for an example configuration.
+
+Please note that this configuration file only contains the essentials.
+Detailed configuration as, i.e., the users, schedule, or channels, will be configured through Icinga Notifications Web.
+
+## Top Level Configuration
+
+### Listener Configuration
+
+The HTTP listener can be used both for submission and for debugging purposes.
+
+| Option         | Description                                                          |
+|----------------|----------------------------------------------------------------------|
+| listen         | Address to be bound to.                                              |
```

Comment:
"Address to bind to" sounds better, IMHO. Also, address is unclear whether this includes a port, so an example would be nice.